### PR TITLE
mind transfer file maintenace i guess okay

### PR DESCRIPTION
**Pull request by gbasood**
<logging>

**THIS IS NOW JUST A CLEANUP PR**

### CODE REVIEW

**Comment by MadmanMartian** [comment]
I'd remove it entirely tbh. I recast it so that ghosts can have minds, so that's a legacy error catcher

**Comment by PJB3005** [comment]
Where's Carn, we need to inform her.

**Comment by gbasood** [comment]
Alright, does that mean that I should also change these?
```
var/mob/living/current
var/mob/living/original
```

**Comment by MadmanMartian** [comment]
https://github.com/vgstation-coders/vgstation13/pull/24113

We want ghosts to receive minds so they can actually be pinged by an admin, and for attack logs to persist. Mind is a /mob level var, not sure why Carn wouldn't want ghosts to have mind

**Comment by gbasood** [comment]
I don't know but I remember that weird distinction led to a lot of FUN bugs because of special code

**Comment by PJB3005** [comment]
Wait what. Why the hell does ahelp need a mind to exist.

That seems like the real bug here.

**Comment by gbasood** [comment]
I undid the original commit and removed references to mob/living in this file

File: code/datums/mind.dm [hunk 1]
```diff
@@ -111,7 +105,7 @@
 	if (hasFactionsWithHUDIcons())
 		update_faction_icons()
 
-/datum/mind/proc/transfer_to_without_current(var/mob/living/new_character)
+/datum/mind/proc/transfer_to_without_current(var/mob//new_character)
```
Comment (jknpj) [inline code/datums/mind.dm]: //

**Comment by jknpj** [comment]
```code/datums/gamemode/role/cultist.dm:104:error: antag.current.adjustBruteLoss: undefined proc```

**Pull request merged**